src/coreComponents/constitutive/fluid/multifluid/MultiFluidBase.cpp

```diff
@@ -37,16 +37,16 @@ MultiFluidBase::MultiFluidBase( string const & name, Group * const parent )
 
   registerWrapper( viewKeyStruct::componentNamesString(), &m_componentNames ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "List of component names" );
+    setDescription( "List of fluid components (e.g. CH4, H2O, C5H12)" );
 
   registerWrapper( viewKeyStruct::componentMolarWeightString(), &m_componentMolarWeight ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Component molar weights" );
+    setDescription( "Molar weights of components" );
 
   registerWrapper( viewKeyStruct::phaseNamesString(), &m_phaseNames ).
     setRTTypeName( rtTypes::CustomTypes::groupNameRefArray ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "List of fluid phases" );
+    setDescription( "List of fluid phases (e.g. gas, water, oil)" );
```

Comment:
These obviously weren't required, but I copied the slightly more detailed description from the duplicate registration in `InvariantImmiscibleFluid`.